onnx_caffe2/backend.py

```diff
@@ -240,12 +241,12 @@ def tensor2list(onnx_tensor):
         return c2_op
 
     @classmethod
-    def _create_constant(cls, n, opset_version):
+    def _create_constant(cls, model_helper, n, opset_version):
```

Comment:
it's all a bit confusing API that we can return either of the two. Maybe there's a way to structure it so that each function implements only one of the versions.